config/ignite-log4j2.xml

```diff
@@ -28,6 +28,12 @@
             <PatternLayout pattern="[%d{ISO8601}][%-5p][%t][%c{1}]%notEmpty{[%markerSimpleName]} %m%n"/>
         </Console>
 
+        <File name="CONSISTENCY" fileName="${sys:IGNITE_HOME}/work/log/consistency.log">
```

Comment:
for ignitetest.tests.control_utility.consistency_test.ConsistencyTest.test_logging.ignite_version